template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Areas/IdentityService/Data/Migrations/00000000000000_CreateIdentitySchema.cs

```diff
@@ -1,13 +1,12 @@
 ﻿using System;
 using System.Collections.Generic;
+using Microsoft.AspNetCore.Identity.Service;
 #if (UseLocalDB)
```

Comment:
This isn't user visible & the parentheses aren't technically not required in this case. This is a carry forward of a limitation in the old expression evaluator used in the templates - it only terminated conditions correctly if they were brace matched. The new evaluator doesn't have that limitation, but conditions still seem to be taking this form, following the pattern of the existing conditions in the templates that were added before the changeover.